lib/models/grid.rb

```diff
@@ -0,0 +1,20 @@
+class Grid < ActiveRecord::Base
+
+  self.table_name = 'youthjobs.grid'
+
+  def self.config_from_yaml
```

Comment:
I saw a clever thing that creates a global `$config` variable, so it's accessible from everywhere. I'll look into setting that up so we can reference the connection as `$config['something']['something']`.